blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/serializers/FakeMetricDataGenerator.java

```diff
@@ -96,6 +96,17 @@
         return points;
     }
     
+    public static Points<BluefloodEnumRollup> generateFakeEnumRollupPoints() {
+        Points<BluefloodEnumRollup> points = new Points<BluefloodEnumRollup>();
+        long startTime = 1234567L;
+        for (int i = 0; i < 5; i++) {
```

Comment:
I am a bit confused why we are generating fake enum rollup points. Is this method only being called in Test classes? If so, should we move it to some class in ```src/test/java``` ?